### PR TITLE
add North Dakota

### PR DESCRIPTION
**Pull request by twagner000**

### CODE REVIEW

File: src/states/north_dakota/main.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import re
+from bs4 import BeautifulSoup
+from common import cache_request, to_list
+
+BASE_URL = 'https://vip.sos.nd.gov/CountyAuditors.aspx'
+
+re_phfaxemail = re.compile(
+  r'\s*(?P<phone>\(?\s*\d{3}\s*\)?\s*\d{3}-?\d{4})\s*(?:.*?)?\n'  # ignore extension
```
Comment (tianhuil) [inline src/states/north_dakota/main.py:8]: I never knew this existed.  +1

File: src/states/north_dakota/main.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+def parse_row(row):
+  cols = [col.get_text('\n').strip() for col in row('td')]
+  county = f"{cols[1]} County"
+  phfaxemail = re_phfaxemail.match(cols[3]).groupdict()
+  return {
+    'locale': county,
+    'county': county,
+    'official': cols[2],
+    'phones': to_list(phfaxemail.get('phone')),
```
Comment (tianhuil) [inline src/states/north_dakota/main.py:23]: Do we check (either here or later on) whether we are able to get the fax / email?  It looks like we should always have one.

Comment (twagner000) [reply src/states/north_dakota/main.py:23]: The regex will fail to match if either one is missing, throwing an error.

Comment (tianhuil) [reply src/states/north_dakota/main.py:23]: LGTM

**Pull request merged**